CHANGELOG.md

```diff
@@ -2,12 +2,20 @@
 
 ## Unreleased
 
-Full list of differences found in [this compare](https://github.com/open-telemetry/opentelemetry-proto/compare/v1.1.0...main).
+Full list of differences found in [this compare](https://github.com/open-telemetry/opentelemetry-proto/compare/v1.3.2...main).
```

Comment:
```suggestion
The full list of changes can be found in the compare view for the respective release at <https://github.com/open-telemetry/opentelemetry-proto/releases>.
```

Otherwise markdown-link-check will block the PR since the tag won't exist yet 🙂